components/MashCards/InitialDecisionCard.tsx

```diff
@@ -0,0 +1,61 @@
+import s from './MashCard.module.scss';
+
+interface Props {
+  clientname: string;
+  timeleft: string;
+  datetime: string;
+  screeningDecision: string;
+  referralCategory: string;
+}
+const InitialDecisionCard = ({
+  clientname,
+  timeleft,
+  datetime,
+  screeningDecision,
+  referralCategory,
+}: Props): React.ReactElement | null => {
+  console.log(screeningDecision);
+  return (
+    <>
+      <li className={s.row}>
+        <div>
+          <p className="lbh-body-m govuk-!-margin-bottom-3">
+            <span className="govuk-tag lbh-tag lbh-tag--green">
+              {timeleft} left
+            </span>{' '}
+            {datetime}
+            <span
+              className="lbh-body-l lbh-!-font-weight-bold"
+              style={{ float: 'right', display: 'block' }}
```

Comment:
Like other comment move this into css class

Reply:
Turned this into a class called 'action' in 'MashCard.module.scss'